x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -22,9 +22,10 @@ export const allowedExperimentalValues = Object.freeze({
   pendingActionResponsesWithAck: true,
   policyListEnabled: true,
   policyResponseInFleetEnabled: true,
+  threatIntelligenceEnabled: false,
```

Comment:
not sure I was allowed to use this object but it seemed to make sense, as the Threat Intelligence plugin is loaded within the Security Solution plugin